docs/settings/alert-action-settings.asciidoc

```diff
@@ -58,3 +58,16 @@ You can configure the following settings in the `kibana.yml` file.
 ==== Alert settings
 
 You do not need to configure any additional settings to use alerting in {kib}.
+
+[float]
+[[event-log-ilm]]
+==== Event log index lifecycle managment
+
+Alerts and actions log their activity in a set of "event log" indices.  These indices are configured with an index lifecycle managment (ILM) policy which can be customized.  The default policy will perform a rollover of the index when it reaches 50GB in size, or after 30 days.  Indices over 90 days old will be deleted.
+
+The name of the index policy is `kibana-event-log-policy`.  It will be created when Kibana starts, if it doesn't exist, otherwise it will never be modified by Kibana.  It can be customized for your environment.
+
+The event log is current used to display historic data on alert and action activity, and for diagnostic purposes.
```

Comment:
```suggestion
The event log displays historic data on alert and action activity, for diagnostic purposes.
```

Reply:
The wording on this is not quite right, as the event log doesn't "display" anything itself.  It just contains the data that Kibana uses to display historic information in the UI.  

I think we can also drop the "diagnostic purposes" part of this - my bad.  I have some separate "how to use the event log for diagnostic purposes" info I'm going to be putting in the event log README.md for now (not quite ready for "production").  I was hoping to drop a hint that "there's some interesting stuff in this index!", but that's just teasing without more info.

Here's a new take:

----

Because the documents in the event log are used to display historic alert and action activity in {kib}, care should be taken when using the ILM delete phase.  For example, if the rolled over indices are deleted after 2 weeks, no historic data will be shown in {kib} that is more than 2 weeks old.

Reply:
Nice!  Suggest changing "care should be taken when" to "take care when".